scene/gui/text_edit.h

```diff
@@ -511,7 +511,7 @@ class TextEdit : public Control {
 
 	/* Syntax highlighting. */
 	Ref<SyntaxHighlighter> syntax_highlighter;
-	Map<int, Dictionary> syntax_highlighting_cache;
```

Comment:
`syntax_highlighting_cache` can be removed, as it's been moved to `SyntaxHighlighter`.

Reply:
Fixed.